Lesson23/Mesut/filter.js

```diff
@@ -269,12 +281,19 @@ const productsExercise6 = [
 	},
 ];
 
+const productsExercise6Result= productsExercise6.filter(product=>product.quantity>0);
+console.log(productsExercise6Result);
+console.log("---------Exercise 7--------------");
 // Exercise 7:
 // Given an array of numbers, filter out the numbers that are divisible by 3.
 const numbersExercise7 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+const numbersExercise7Result =numbersExercise7.filter(num=>num%3!==0)
```

Comment:
Here again you need to filter out values divisible by 3, so negation of the result is needed. Also, it's always better to use a strict comparison ('===').
```suggestion
const numbersExercise7Result =numbersExercise7.filter(num=>!num%3! !== 0)
```